packages/fxa-settings/tailwind.config.js

```diff
@@ -29,4 +29,45 @@ config.theme.extend.backgroundImage = {
   'ff-logo': "url('../../../fxa-react/images/ff-logo.svg')",
 };
 
+config.theme.extend.keyframes = {
```

Comment:
I think you're going to want to spread the existing keyframes like we are for `backgroundImage` so they don't get overwritten from the original config.

Also not necessary, but I think I prefer everything under `config.theme.extend` to be in one object kind of like what we're doing in fxa-content-server's Tailwind config file, but it's not a strong preference.